third_party/winuwp_h264/H264Encoder/H264Encoder.cc

```diff
@@ -95,50 +157,10 @@ int WinUWPH264EncoderImpl::InitEncode(const VideoCodec* codec_settings,
     target_bps_ = width_ * height_ * 2;
```

Comment:
>width_ * height_ * 2 [](start = 18, length = 20)

This doesn't pass the smell test.... Do you know if HL2 code takes this branch? Have you tried to change the constant 2?

Reply:
In the experiments I have done with the test app I have only seen the second branch (startBitrate) taken. The expression is indeed a bit dumb but it is only used as a starting value, the webrtc feedback loop will change it quickly. So good enough IMHO.